pkg/rtc/participant.go

```diff
@@ -707,8 +707,7 @@ func (p *ParticipantImpl) GetConnectionQuality() livekit.ConnectionQuality {
 		if subTrack.IsMuted() {
 			continue
 		}
-		// LK-TODO: maybe this should check CurrentSpatialLayer as target may not have been achieved
-		if subTrack.DownTrack().TargetSpatialLayer() < subTrack.DownTrack().MaxSpatialLayer() {
+		if !subTrack.DownTrack().IsForwardingOptimal() {
```

Comment:
@davidzhao I am pulling this as an API into `sfu.DownTrack`. Still the same formula, but inside down track now. Still undecided whether we should compare against current or target.